src/styles/index.scss

```
@@ -1,3 +1,79 @@
 body {
+  height: 100vh;
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch__circle {
+  position: absolute;
+  width: 5vmin;
+  height: 5vmin;
+  background-color: #f6a603;
+  border-radius: 50%;
+  z-index: 10;
+}
+
+.stopwatch__minute-hand {
+  position: absolute;
+  width: 3vmin;
+  height: 20vmin;
+  background-color: #0700ff;
+  transform-origin: bottom;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -100%) rotate(0deg);
+  animation: minute-rotation 3600s steps(60) infinite;
+  z-index: 1;
+}
+
+.stopwatch__second-hand {
+  position: absolute;
+  width: 1.5vmin;
+  height: 38vmin;
+  background-color: #2c8000;
+  transform-origin: bottom;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -100%) rotate(0deg); /* Начальная позиция */
+  animation: second-rotation 60s linear infinite;
+  z-index: 1;
+}
+
+@keyframes second-rotation {
+  0% {
+    transform: translate(-50%, -100%) rotate(0deg);
+  }
+  100% {
+    transform: translate(-50%, -100%) rotate(360deg);
+  }
+}
+
+@keyframes minute-rotation {
+  0% {
+    transform: translate(-50%, -100%) rotate(0deg);
+  }
+  100% {
+    transform: translate(-50%, -100%) rotate(360deg);
+  }
+}
```

Don't need to duplicate key frames with the same scenario
```suggestion
@keyframes rotation {
  0% {
    transform: translate(-50%, -100%) rotate(0deg);
  }
  100% {
    transform: translate(-50%, -100%) rotate(360deg);
  }
}
```